openpype/hosts/houdini/plugins/create/create_unreal_staticmesh.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating Unreal Static Meshes.
+
+Unreal Static Meshes will be published as FBX.
+
+Filmbox by default expects an ObjNode
+however, we set the sop node explictly
+to eleminate any confusion.
+
+This will make Filmbox to ignore any object transformations!
```

Comment:
If forcing a `SopNode` over `ObjNode` causes 'lack of features' in cases where one might WANT those transformations we might want to allow that?

Reply:
I ran out of ideas tbh. 

How can we do some validations without specifying the output node explicitly 
to be specific "path" attribute 
I keep think about how to handle collision prefixes

Reply:
Just want to clear up some of my confusion, what kind of collision prefixes you want to collect before exporting the fbx to Unreal? Are you talking about LOD(Level of Details) or some animation data? If you talk about LODs, I am not sure about the LODs of the filmbox fbx works the same with the HDA. All I know is Sidefbx suggests to export the piece of LODs data to HDA and import the HDA to Unreal. You can find the doc here: https://www.sidefx.com/docs/houdini/unreal/collisionsetup.html#lod
If you talk about animation data, I dont think you need that for static mesh. You can always set up collision data with static mesh in Unreal. 
Or are you talking about axis issue? We need to set the Y Axis up for unreal if the axis is not right(of which Z axis goes up).

Reply:
> what kind of collision prefixes

I mean that, meshes can't start with these name 
and I think I have two options 
A. Check Object names (I'm going to try it) 
B. Check Path attribute (can be a pain)

![image](https://github.com/ynput/OpenPype/assets/20871534/c74aa519-7351-438b-a665-2988d7d1b1a3)